components/eam/src/physics/crm/samxx/vars.cpp

```diff
@@ -1652,6 +1682,9 @@ real2d t_vt_tend      ;
 real2d q_vt_tend      ;
 real4d t_vt_pert      ;
 real4d q_vt_pert      ;
+real2d u_vt           ;
+real2d u_vt_tend      ;
+real4d u_vt_pert      ;
```

Comment:
here too. you can reorder these lines so that u_vt comes after q_vt, and u_vt_tend comes after q_vt_tend, and u_vt_pert comes after q_vt_pert.

Reply:
fixed